src/test/java/core/basesyntax/strategy/FruitHandlerStrategyTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.enumeration.Operation;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.impl.BalanceFruitHandlerImpl;
+import core.basesyntax.strategy.impl.PurchaseFruitHandlerImpl;
+import core.basesyntax.strategy.impl.ReturnFruitHandlerImpl;
+import core.basesyntax.strategy.impl.SupplyFruitHandlerImpl;
+import java.util.HashMap;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitHandlerStrategyTest {
+    private static FruitHandlerStrategy fruitHandlerStrategy;
+    private static FruitTransaction fruitTransaction;
```

Comment:
this fruitTransaction could be left as a field, but let's not make it static as static variables are shared between all test methods (and unexpected results could appear)